### ISSUE TITLE
Can't run Steam on Fedora Atomic Workstation due to /home symlink

### ISSUE THREAD

**Issue by jlebon** [flatpak/flatpak#1280]
This looks like a regression of https://github.com/flatpak/flatpak/issues/493, but specifically for Steam. I'm making it a separate issue because interestingly, `org.gnome.Evince` does work (unlike in #493).

```
$ rpm-ostree status
State: idle
Deployments:
● faw27:fedora/27/x86_64/workstation
                   Version: 27.22 (2017-12-17 19:43:47)
                BaseCommit: 2ace750cd18be4c21e7342a1876f70f15cefc56a1793ff8afa81a3a0b107631e
              GPGSignature: Valid signature by 860E19B0AFA800A1751881A6F55E7430F5282EE4
      ReplacedBasePackages: docker-rhel-push-plugin docker docker-common 2:1.13.1-42.git4402c09.fc27 -> 2:1.13.1-44.git584d391.fc27
           LayeredPackages: chrome-gnome-shell krb5-workstation libvirt-client mosh oci-kvm-hook origin-clients sshpass strace tmux vim vim-X11 virt-install virt-manager xsel

$ rpm -q flatpak
flatpak-0.10.1-1.fc27.x86_64

$ strace -f -e symlink,symlinkat flatpak run com.valvesoftware.Steam/x86_64/stable
strace: Process 15464 attached
strace: Process 15465 attached
strace: Process 15466 attached
strace: Process 15467 attached
[pid 15466] +++ exited with 0 +++
[pid 15463] --- SIGCHLD {si_signo=SIGCHLD, si_code=CLD_EXITED, si_pid=15466, si_uid=1000, si_status=0, si_utime=0, si_stime=0} ---
strace: Process 15468 attached
[pid 15468] symlink("var/mnt", "/newroot/mnt") = 0
[pid 15468] symlink("var/srv", "/newroot/srv") = 0
[pid 15468] symlink("var/opt", "/newroot/opt") = 0
[pid 15468] symlink("run/media", "/newroot/media") = 0
[pid 15468] symlink("usr/lib", "/newroot/lib") = 0
[pid 15468] symlink("var/home", "/newroot/home") = 0
[pid 15468] symlink("usr/bin", "/newroot/bin") = 0
[pid 15468] symlink("usr/lib64", "/newroot/lib64") = 0
[pid 15468] symlink("usr/sbin", "/newroot/sbin") = 0
[pid 15468] symlink("var/roothome", "/newroot/root") = 0
[pid 15468] symlink("sysroot/ostree", "/newroot/ostree") = 0
[pid 15468] symlink("sysroot/tmp", "/newroot/tmp") = 0
strace: Process 15469 attached
[pid 15465] +++ exited with 0 +++
[pid 15464] +++ exited with 0 +++
strace: Process 15470 attached
[pid 15470] symlink("../run", "/newroot/var/run") = 0
[pid 15470] symlink("/usr/lib/GL/glvnd", "/newroot/etc/glvnd") = 0
[pid 15470] symlink("/proc/mounts", "/newroot/etc/mtab") = 0
[pid 15470] symlink("../usr/lib/os-release", "/newroot/etc/os-release") = 0
[pid 15470] symlink("/usr/lib/GL/vulkan", "/newroot/etc/vulkan") = 0
[pid 15470] symlink("usr/lib", "/newroot/lib") = 0
[pid 15470] symlink("usr/lib64", "/newroot/lib64") = 0
[pid 15470] symlink("usr/bin", "/newroot/bin") = 0
[pid 15470] symlink("usr/sbin", "/newroot/sbin") = 0
[pid 15470] symlink("/run/host/monitor/localtime", "/newroot/etc/localtime") = 0
[pid 15470] symlink("/run/host/monitor/resolv.conf", "/newroot/etc/resolv.conf") = 0
[pid 15470] symlink("/run/host/monitor/host.conf", "/newroot/etc/host.conf") = 0
[pid 15470] symlink("/run/host/monitor/hosts", "/newroot/etc/hosts") = 0
[pid 15470] symlink("../../../.flatpak-info", "/newroot/run/user/1000/flatpak-info") = 0
[pid 15470] symlink("var/home", "/newroot/home") = -1 EEXIST (File exists)
bwrap: Can't make symlink at /home: File exists
[pid 15470] +++ exited with 1 +++
[pid 15469] +++ exited with 0 +++
[pid 15463] +++ exited with 1 +++
[pid 15468] +++ exited with 0 +++
+++ exited with 0 +++
```

One can work around this by setting `HOME=/var/home/jlebon flatpak run ...`.

**Comment by mczimmermann** [comment]
This is the same issue as #1278. 

Thank you for providing a workaround =)

**Comment by alexlarsson** [comment]
closed as dup of  #1278
